### ISSUE TITLE
[Voice Access] [Startup: Launch Size]: No number is shown for the 'x' button inside the startup page by 'Show numbers' command of Voice Access.

### ISSUE THREAD

**Issue by Shubham786786** [microsoft/terminal#14399]
### Windows Terminal version

1.17.3052.0

### Windows build number

Windows 11 Version 22H2 (OS Build 25206.1000)

### Steps to reproduce

1. Open Windows Terminal.
2. Open 'Settings > Startup' window with 'Ctrl+,' keys.
3. Open Voice Access by 'Settings > Accessibility > Speech > Turn on Voice Access.
4. Navigate to 'Launch size' button by tab key and expand it.
5. Give command as 'Show numbers' by Voice Access.
6. Observe the issue.

**User Experience: -**
Voice Access users will not be unable to access icon button 'x', if these are not accessible using 'Show numbers' commands of VA.

**Guideline Reference:**
https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/04.03.01

**Attachments:**
[No number is shown for the 'x' button inside the startup page by 'Show numbers' command of Voice Access..zip](https://github.com/microsoft/terminal/files/10019968/No.number.is.shown.for.the.x.button.inside.the.startup.page.by.Show.numbers.command.of.Voice.Access.zip)

### Expected Behavior

Number should be shown for the 'x' button present inside the 'Launch Size' edit box by 'Show numbers' command of Voice Access.

https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fwww.osgwiki.com%2fwiki%2fVoiceAccess_Testing&data=05%7c01%7cv-chisha%40microsoft.com%7cad1aa7ab3c63420978a708da31b8e712%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637876967593170034%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000%7c%7c%7c&sdata=vx8nL3zR6xV5J7ImC%2BkIAygjzuu5wOXDwim31oXAtpc%3D&reserved=0

### Actual Behavior

No number is shown for the 'x' button present inside the 'Launch Size' edit box by 'Show numbers' command of Voice Access.

Similar is the case for the 'Rows' 'x' button.

**Comment by carlos-zamora** [comment]
This is a WinUI issue as the bug can be reproduced in the WinUI 2 Gallery.
I've filed this issue upstream here: https://github.com/microsoft/microsoft-ui-xaml/issues/7996

Tagging as tracking external.

CC @Shubham786786

**Comment by DHowett** [comment]
This is a WinUI bug. We are using a WinUI NumberBox for this control.

**Comment by Priyanka-Chauhan123** [comment]
Closing as external to WinUI. Bug has been shared with WinUI for getting its tracking ID.

**Comment by Priyanka-Chauhan123** [comment]
[Bug 36874815](https://microsoft.visualstudio.com/OS/_workitems/edit/36874815): which has filed against Win UI and closed it as By Design.
